tests/testthat/test-ensemble.R

```diff
@@ -53,59 +54,54 @@ test_that("We can extract resdiuals from caretEnsemble objects", {
   expect_equal(length(r), 150)
 })
 
-test_that("We can extract resdiuals from train objects", {
+test_that("We can extract resdiuals from train regression objects", {
   data(iris)
   mod <- train(
-    iris[, 1:2], iris[, 3], method="lm",
-    trControl=trainControl(
-      method="cv", number=3, savePredictions="final")
+    iris[, 1:2], iris[, 3],
+    method = "lm",
+    trControl = trainControl(
+      method = "cv", number = 3, savePredictions = "final"
     )
+  )
   r <- residuals(mod)
   expect_is(r, "numeric")
   expect_equal(length(r), 150)
-
-  #NOT YET SUPPORTED
-#   y <- factor(ifelse(iris[,5]=="versicolor", "yes", "no"))
-#   mod <- train(iris[,1:2], y, method="glm", trControl=trainControl(method="cv", number=3, savePredictions=TRUE))
-#   r <- residuals(mod)
```

Comment:
caret.residuals doesn't support classification models (lets update it sometime so it does, and while we're add it save class probs by default)